### ISSUE TITLE
Remove or fix ignored scala tests

### ISSUE THREAD

**Issue by laneb** [TresAmigosSD/SMV#853]

**Comment by laneb** [comment]
`ClassCRCTest` can't run in Maven, so this issue can't be completed until we drop support for Maven.

**Comment by AliTajeldin** [comment]
@laneb do we know the root cause of failure on maven?

**Comment by laneb** [comment]
From `ClassCRCTest.scala`:

```
  // tests that involve compilers can run in sbt, but not in mvn via scalatest-mvn plugin
  // with a NoSuchMethodError thrown in scala.reflect
  // temporarily ignore the 2 tests
```

**Comment by laneb** [comment]
Also, the fix for `GraphUtilTest` needs to be merged from Spark 2.1 branch. This issue will be complete once `GraphUtilTest` and `ClassCrcTest` are resolved.

**Comment by ninjapapa** [comment]
Not an issue anymore.
